usaspending_api/api_contracts/contracts/v2/federal_accounts/available_object_classes.md

```diff
@@ -0,0 +1,27 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Available Object Classes [v2/federal_accounts/{federal_account_id}/available_object_classes]
+
+## GET
+
+This route returns object classes that the specified federal account has allotted money to.
+
++ Parameters
+    + `federal_account_id`: 5 (required, number)
+        Database id for a federal account.
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + `results` (required, array[MajorObjectClass], fixed-type)
+
```

Comment:
Can the JSON from the api doc be included in this doc as the example response body?